src/style.css

```diff
@@ -0,0 +1,94 @@
+/* #region general */
+@font-face {
+  font-family: "Avenir-Book";
+  src: local("Avenir-Book.ttf");
+  font-weight: 300;
+}
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: local("Avenir-Heavy.ttf");
+  font-weight: 300;
+}
+/* #endregion */
+
+/* #region search-bar */
+.search-bar {
+  position: relative;
+  box-sizing: border-box;
+  margin-top: 20px;
+  border-radius: 4px;
+}
+
+.search-bar::before {
+  position: absolute;
+  content: "";
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: 50%;
+}
+
+.search-bar--size--big::before {
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.search-bar--size--small::before {
+  height: 11px;
+  width: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.search-bar--size--big:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.search-bar--size--big:focus {
+  outline: none;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-family: "Avenir-Heavy", "Arial", sans-serif;
+}
+
+.search-bar--size--small:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.search-bar--size--small:focus {
+  outline: none;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-family: "Avenir-Heavy", "Arial", sans-serif;
+}
```

Comment:
DRY. We have two identical states for different sizes, can we do the same for the general class `search-bar`?

Reply:
Not fixed. You have same styles on focus for big and small search bars. Just add them to default search bar on focus
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/96409563/d0d9e439-8a0f-4f09-91c8-6cc8d73e1a01)